object/user_cred_test.go

```diff
@@ -24,8 +24,8 @@ func TestSaltedPasswordsGeneration(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			organization := Organization{PasswordType: tt.name}
-			user1.UpdateUserPassword(&organization)
-			user2.UpdateUserPassword(&organization)
+			_ = user1.UpdateUserPassword(&organization)
```

Comment:
if UpdateUserPassword return error now - lets handle it, in logs atleast

Reply:
should we handle it here? the test aims to verify password hashing, not error handling